### ISSUE TITLE
Mage Characters do not start Friendly with the Kirin Tor faction

### ISSUE THREAD

**Issue by Feloxis94** [WoWManiaUK/Redemption#2277]
Hello, i realize this issue when i came to Dalaran for the first time with my blood elf mage to see that the Kirin Tor faction were neutral. Quite strange considering that  im a mage lol  and that on retail  wow back in the day all mages started friendly with the Kirin Tor.

**Comment by hotdoggidy** [comment]
You may want to find some proof of this to help expedite. Perhaps some patch notes or a retail video showing what you are referring to.

**Comment by Dranah** [comment]
May i ask what race and faction do you play?
My blood elf mage started as friendly as intended.

**Comment by Feloxis94** [comment]
https://wow.gamepedia.com/Mage#Dalaran

click on the Trivia section, it says it there.

**Comment by Dranah** [comment]
did you create your character recently?

**Comment by Feloxis94** [comment]
I create it,more than a month ago, a blood elf fire mage

**Comment by Dranah** [comment]
Newly created mages now properly receive their Kirin Tor Rep to friendly when enterring northrend.
Old ones that were created during the bugged period still stay on neutral, may need to contact a gm (or just farm a little more rep)

**Comment by Dranah** [comment]
Something has been cucked here.
TL;DR: mages made post this fix have somehow gained +3000 rep ,at least visually. Some complain on Red that certain actions 'take back' this rep, and they suddenly lost3k.
This wasnt in place when I confirmed fix, new mages gained 1/6000 friendly status. Now they gain 3000/6000, no idea why.
Mages created before the bug are not affected at all and mages created during the neutral bug period are still 0/3000 neutral.

Detailed:
I have a play toon mage created in 2017 before the bug, her rep status is unaffected (as are all other very old mages)
I have a test mage created in 2018 january (after bug, but before fix), with 0/3000 neutral rep as she was made while the native rep gain was broken. Her rep standing did not change since, still 0/3000.
My 2nd test mage made 2018 march (post-fix), had proper 1/6000 friendly rep when I made him, but now, he has 3000/6000 rep for no reason, I have not farmed any rep on him so his status should be still 1/6000. Not sure when exactly it happened, I wasn't checking his rep stats for a while.
Creating any new mages now also results in a 3000/6000 friendly rep status , which is 3000 more than intended.
GMs on redemption been receiving reports from mages suddenly losing 3k rep out of the blue. I could not reproduce this yet, but it leads me to believe the 3k extra rep is merely visual and some actions set it back to show the intended value.

**Comment by Dranah** [comment]
New mages now start with 1000/3000 neutral. It's a change, but not quite what we need yet.
Need to start with 1/6000 friendly.
Older mages' rep status unaffected.

**Comment by wow-mania** [comment]
This 100% fixed in dev I have checked.

**Comment by Dranah** [comment]
![WoWScrnShot_040319_212605](https://user-images.githubusercontent.com/31132743/55507027-1ff7fc80-5657-11e9-9389-01106fe415a9.jpg)
I just made a new mage, you can see half minute played time...
1000/3000 neutral.
how the heck is it fixed????

**Comment by wow-mania** [comment]
![image](https://user-images.githubusercontent.com/12248683/55537551-616ac500-56b4-11e9-977e-b4c1d24689fc.png)
Simple...
It's fixed for all new characters. How the heck can we fix for existing character? :)

**Comment by Dranah** [comment]
...
I have clearly written that i made a NEW char, i do not check existing ones because i know their rep cannot be modded.
Any new mage I create starts with 1000/3000 neutral. 
Text even says 
"You are now Neutral with the Kirin Tor."
"Reputation with Kirin Tor increased by 1000."
![WoWScrnShot_040419_113917](https://user-images.githubusercontent.com/31132743/55546043-c3382880-56ce-11e9-8ddd-61cc26952e5e.jpg)

What are we doing differently?

**Comment by Dranah** [comment]
Seems the visibility only works on the next patch which is not yet released.
Mages will see wrong rep displayed until the client update.
Mages created before the fix need to contact a GM to have their rep fixed.
